### PR TITLE
Seperatecomponent

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
Closes: #199 #200 

@Namanl2001 please review this.

### CODE REVIEW

File: src/components/filter.jsx [hunk 1]
```diff
@@ -130,7 +130,16 @@ class filter extends Component {
               <Dropdown.Item onClick={this.props.c12}>12 th</Dropdown.Item>
             </Dropdown.Menu>
           </Dropdown>
-
+        </div>
+        <div
+          style={{
```
Comment (Namanl2001) [inline src/components/filter.jsx:135]: The styling here and at line 70 seems to be same. Would be nice if we can have this once in style.css file.

**Review by Namanl2001 — COMMENTED** [review]
Small nit.

**Comment by ChitvanRamani22** [comment]
@Namanl2001 done.

**Review by Namanl2001 — APPROVED** [review]
LGTM Thanks

**Pull request merged**